clusterMembership/staticCluster.go

```diff
@@ -11,46 +8,19 @@ import "errors"
  * Keeps a Map of the nodeId to Node for fast lookup of nodes.
  */
 type staticCluster struct {
-	members    []string
-	membersMap map[string]Node
+	members []Node
 }
 
-func (c staticCluster) Members() []string {
+func (c staticCluster) Members() []Node {
 	return c.members
```

Comment:
It annoys me that we end up returning mutable pointers to our internal data. This is the idiomatic way to do it, just, mentioning that it bugs me each time.